src/os/bluestore/BlueFS.cc

```diff
@@ -91,6 +91,9 @@ class BlueFS::SocketHook : public AdminSocketHook {
 	r = admin_socket->register_command("bluefs files list", hook,
 					   "print files in bluefs");
 	ceph_assert(r == 0);
+	r = admin_socket->register_command("bluefs inject_read_zeros", hook,
```

Comment:
add a note that this is testing/debug command only?

Reply:
Done.